docs/user-guide/api-mediation/api-mediation-multi-tenancy.md

```diff
@@ -60,6 +60,13 @@ components.gateway.apiml.service.additionalRegistration:
               - gatewayUrl: /
                 serviceUrl: /
 ```
+
+```
+components.gateway.apiml.security.x509:
+    #  cloud gateway port 
+        certificatesUrl: https://sys1:10018/gateway/certificates
```

Comment:
I would have it similar like in other articles, w/o specific port and host. Something like this:
```suggestion
        certificatesUrl: https://{cloudGatewayHost}:{cloudGatewayPort}/gateway/certificates
```

Reply:
Ok, it's an example actually. Then it's fine.